spacemesh/v2alpha1/smeshing_identities.proto

```diff
@@ -30,11 +31,56 @@ enum IdentityState {
   PROPOSAL_PUBLISHED = 12;
 }
 
+message RetryingState {
+  string message = 1;
+}
+message WaitForAtxSyncedState {}
+message WaitingForPoetRegistrationWindowState {}
+message PoetChallengeReadyState {}
+
+message PoETRegistration {
+  bytes challenge_hash = 1;
+  string address = 2;
+  string round_id = 3;
+  google.protobuf.Timestamp round_end = 4;
+}
+message PoetRegisteredState {
+  repeated PoETRegistration registrations = 1;
+}
+
+message WaitForPoetRoundEndState {}
```

Comment:
Perhaps it could also include a timestamp of the round end?

Reply:
Done